pages/03.content/02.blog/_bios/copeet/bio.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Connor Peet
+image: https://avatars1.githubusercontent.com/u/2230985?s=128
+social:
```

Comment:
We can probably store these variables in the user configuration.

Reply:
Eh. Having them as content-only lets people who don't have grav admin access author blog posts. Maybe we have guest days with posts written by 3rd party developers for instance :)

Reply:
We can give guests admin access, there's a full permission system.

Reply:
Sure